river/tree/arf_hoeffding_tree_classifier.py

```diff
@@ -97,4 +97,19 @@ def _new_learning_node(self, initial_stats=None, parent=None):
             return RandomLearningNodeNBA(initial_stats, depth, self.max_features, seed)
 
     def new_instance(self):
-        return self.__class__(**self._get_params())
+        return self.__class__(max_size=self.max_size,
+                              memory_estimate_period=self.memory_estimate_period,
+                              grace_period=self.grace_period,
+                              split_criterion=self.split_criterion,
+                              split_confidence=self.split_confidence,
+                              tie_threshold=self.tie_threshold,
+                              binary_split=self.binary_split,
+                              stop_mem_management=self.stop_mem_management,
+                              remove_poor_attrs=self.remove_poor_attrs,
+                              merit_preprune=self.merit_preprune,
+                              leaf_prediction=self.leaf_prediction,
+                              nb_threshold=self.nb_threshold,
+                              nominal_attributes=self.nominal_attributes,
+                              max_features=self.max_features,
+                              max_depth=self.max_depth,
+                              seed=self._rng)
```

Comment:
Was not it possible to devise a more automated solution with variants of `_get_params`?

Reply:
I tried but it seems that `**kwargs` are trickier to retrieve. I suggest we use this solution for the time being while we devise a solution that can be applied to any estimator (also requiered for a `clone` method).

Reply:
I agree. This is just a minor detail :)